draftable/test_client.py

```diff
@@ -185,3 +203,34 @@ def test_create_retrieve_export_coverpage(comparisons, exports):
             break
 
     comparisons.delete(comparison.identifier)
+
+
+def test_create_retrieve_change_details(comparisons):
+    comparison = comparisons.create(
+        left="test-files/hello-left.rtf",
+        right="test-files/hello-right.rtf",
+        expires=datetime.datetime.now() + datetime.timedelta(days=1),
+    )
+    assert (
+        not comparison.ready
+    ), "We do not expect the comparison to be ready yet"
+    assert not comparison.failed
+
+    while True:
+        comparison = comparisons.get(comparison.identifier)
+        if comparison.ready:
+            assert not comparison.failed
+            break
+
+    change_details = comparisons.change_details(comparison.identifier)
+
+    with open("test-files/change-details.json", "r") as f:
+        expected_change_details = json.load(f)
+
+    assert change_details.to_dict() == expected_change_details
```

Comment:
the exact content of changes json is subject to change outside of this library. I am changing the assert to something less strict.